src/diffusers/models/attention.py

```diff
@@ -1,10 +1,29 @@
 import math
-from typing import Optional
+import os
+from inspect import isfunction
+from typing import Any, Optional
 
 import torch
 import torch.nn.functional as F
 from torch import nn
 
+import xformers
```

Comment:
This would currently break `pip` - maybe we can make it a "soft-dependency"! Will check it out :-)